content/articles/stream-filter-in-java-using-lambda-expressions/index.md

```diff
@@ -173,5 +185,6 @@ List employeesWithValidProfilePicture = employee
 ```
 
 ### Conclusion
-The filter() method has been introduced to you in this lesson as a way to handle streams. Apart from that, we looked at different approaches to dealing with exceptions. To learn more about Java Stream Filter using Lambda Expression, this article is very valuable.
+
+In this lesson, you have been introduced to the filter() method as a way to handle streams. Apart from that, we looked at different approaches to dealing with exceptions. To learn more about Java Stream Filter using Lambda Expression, this article is very valuable.

```

Comment:
> To learn more about Java Stream Filter using Lambda Expression, this article is very valuable.

This sentence is very awkward here. It sounds like you are about to provide them with more resources to learn about this topic but you just tell them that the article is valuable in teaching this topic. I would either remove this sentence or change it to give further resources about this topic because this sentence as it is is unnecessary.